src/state/markets/test_market.rs

```diff
@@ -33,6 +34,13 @@ fn setup_market_with_params(
     *dex
 }
 
+fn get_clock_fn() -> (u64, u64) {
+    let now = SystemTime::now();
+    let since_the_epoch = now.duration_since(UNIX_EPOCH).expect("Time went backwards");
+    let in_ms = since_the_epoch.as_millis();
+    (in_ms as u64, since_the_epoch.as_secs())
```

Comment:
don't like using ms even though it's the test, can we just hardcode a fake realistic slot number?

Reply:
This is the placeholder, I can replace with a noop